rpc/handlers.go

```diff
@@ -50,6 +51,35 @@ func (s *rpcServer) GetV1ConsensusState(ctx echo.Context) error {
 	})
 }
 
+func (s *rpcServer) GetV1QueryNodeParams(ctx echo.Context) error {
+	currHeight := s.GetBus().GetConsensusModule().CurrentHeight()
+	persistenceRC, err := s.GetBus().GetPersistenceModule().NewReadContext(int64(currHeight))
+	if err != nil {
+		return ctx.String(http.StatusInternalServerError, err.Error())
+	}
+	paramValueMap, err := persistenceRC.GetAllParams()
+	if err != nil {
+		return ctx.String(http.StatusInternalServerError, err.Error())
+	}
+	parameterKeyValues := make([]*paramValue, 0)
+	keys := make([]string, 0)
+	for key := range paramValueMap {
+		keys = append(keys, key)
+	}
+	sort.Strings(keys)
```

Comment:
So the reason I sort here Is because its a map, instead I will try to make a slice like this `{name, value},{name, value}` this might be more efficient